### ISSUE TITLE
BUILD overrides all other flags, unlike in WG 5

### ISSUE THREAD

**Issue by LadyCailinBot** [EngineHub/WorldGuard#977]
WORLDGUARD-3152 - Reported by TNTUP

Summary of issues I found: (NOTE those issues are being tested on a TEST server)
1. BUILD flag overrides everything, no access (players) (Build allow in player's regions), no luck
2. Priority (higher numbers) doesn't work, usually created regions on __global__ already overrides BUILD flag
3. BUILD deny flag invincibles players (mobs, falling, pvp) but in protected regions, and allowed regions. In __global__ without BUILD flag the falling, hit by mobs, pvp is fine
4. Allow door interacting only (?)

OK:

BUILD flag or not allowed (without BUILD flag) cannot interact with doors/entities/chests

1. My server's __global__ has the BUILD deny to prevent illegal building (mostly blocking for mining, I have a mining world!), theres players's own regions, they cannot build/interact/pvp/entity hit/no falling because of the __global__'s BUILD deny. The region __global__ isn't supposed to override player's regions because it has no priority, but player's regions do, they'Re set at 0, I tried setting a higher number, no luck. 

2. I tried higher priorities depite BUILD deny set to __global__ in player's regions, no work. Player's regions priorities, some are set to -1 because they manage their own town/city that theres sub-regions that their pritority is at 0. Player's regions (personal use) priorities are set to 0.

3. The BUILD deny flag also invincibles players to be killed by monsters, falling, pvping etc. Without BUILD deny, everything is fine, but in non allowed regions (theres no BUILD deny!) same issue if BUILD deny were added in that player's region.

4. Doors are automatically blocked by interacting them, couldn't allow those only like as before? (Using LWC for this)

Warning: My english is poor so sorry if I did misspelling or bad grammar. If other issues discovered, ill create another ticket or reply here.
